### ISSUE TITLE
RLS: v0.19.2

### ISSUE THREAD

**Issue by jorisvandenbossche** [pandas-dev/pandas#14748]
Issue to follow-up 0.19.2 release.

0.19.2 milestoned issues/PRs: https://github.com/pandas-dev/pandas/milestone/45

0.19.1 was released Nov 3. In the meantime already quite some work for 0.19.2 is done (mainly thanks to Jeff!). Main things for 0.19.2 (besides general bug fixes):

- Python 3.6 compatibility (https://github.com/pandas-dev/pandas/issues/14679). 
  - This is already done I think thanks to Jeff
- fix segfault with latest dateutil (done in #14631)
- possible regressions (I don't think we have any urgent reported at the moment, only https://github.com/pandas-dev/pandas/issues/14687)

The Python 3.6 release schedule is a rc1 at 2016-12-05, and final release at 2016-12-16. 

Regarding timeline for pandas 0.19.2, the question is maybe whether we have to wait until after the final Python 3.6 release, or if we can do it earlier (eg after rc1, with the risk there are still some changes)

@pandas-dev/pandas-core

**Comment by stonebig** [comment]
hi, would it be possible to have a 0.19.2beta BEFORE rc1, to help check there no more bugs in python-3.6.0b4  itself ?

**Comment by jreback** [comment]
as soon as the latest 3.6 is on conda forge we can build with it - if u want to ping the maintainers

**Comment by jreback** [comment]
so 3.60b4 is now active (e.g. on this PR): https://travis-ci.org/pandas-dev/pandas/jobs/179477285

**Comment by stonebig** [comment]
no errors, wonderfull !

**Comment by jreback** [comment]
@stonebig yeah I don't think any actual changes in the latest betas matters much to us. (I fixed things on 3.60b3)

**Comment by stonebig** [comment]
no waiting that Christoph Gohlke does a binary ... you may perhaps do a beta to trigger it.

**Comment by jreback** [comment]
@stonebig we don't usually do a pre for a minor release.

**Comment by stonebig** [comment]
the idea was to get a wheel. pythonnet projet generate an artefact wheel on every test on appveyor. would be nice of pandas could too

**Comment by jreback** [comment]
@stonebig we generate a tremendous number of PR's / commits. Building the package (via wheel) will just add a lot of time to the build, then have to manage the .whl and such. We are already generating conda packages (though not actually doing anything with them).

To be honest I don't really want to do it because it inevitable causes people to use them, and then complain if they are not super backwards compatible (IOW these are usually generated by a modern compiler / arch and not the compatible layers that say conda-forge / anaconda uses).

So I think people should just build from source if they really really need something in-between relatively frequent versions. Instructions are very clear, and even on windows this i quite easy now a days.

**Comment by stonebig** [comment]
ok, will do a try

**Comment by stonebig** [comment]
falling on second line of procedure:
`````
python setup.py build --compiler=mingw32
`````
that cannot possibly be ok on windows for Python-3.6

**Comment by jreback** [comment]
http://pandas-docs.github.io/pandas-docs-travis/contributing.html#creating-a-windows-development-environment

@stonebig not sure where you are looking, pls following the instructions above, you must use VS 14 to compiile >= 3.5.

**Comment by stonebig** [comment]
I was looking at the front page: https://github.com/pandas-dev/pandas

**Comment by jorisvandenbossche** [comment]
@stonebig That's indeed outdated. I think we can remove almost the full part on installing from source and refer to the docs instead.

**Comment by jorisvandenbossche** [comment]
I updated the 0.19.x branch with all commits tagged for 0.19.2 (only from yesterday I still have to add), and only few things left on the milestone (https://github.com/pandas-dev/pandas/milestone/45)

Python 3.6 release is due for tomorrow. So we can update our travis build matrix to test with the released version.

I would propose to just release one of the coming days, without looking to much to the python 3.6 build artefacts. As it will probably take some time before the conda-forge / wheel-building machinery are updated to include 3.6 packages. 
So I would propose not to wait on that, already release and we can then later upload the 3.6 wheels/conda-packages. Thoughts?

**Comment by jreback** [comment]
agreed with all 

will update the matrix for 3.6 but the upstream will take time to propagate anyhow

**Comment by TomAugspurger** [comment]
FYI, 3.6 was delayed a bit: https://mail.python.org/pipermail/python-dev/2016-December/147000.html Now scheduled for the 23rd.

**Comment by jreback** [comment]
FYI I just updated the matrix. But we are independent of 3.6 releases.

**Comment by jreback** [comment]
thanks again to @jorisvandenbossche for a great release!

**Comment by jorisvandenbossche** [comment]
Just sent out the announcement. Needed follow-up:

- mac/linux wheels for python 3.6: https://github.com/MacPython/pandas-wheels/pull/7#pullrequestreview-14355538 (windows wheels for 3.6 are already there), but that probably has to wait until updates in manylinux/multibuild infrastructure to support 3.6
- conda-forge packages for 3.6: windows OK, linux OK, mac still failing (https://travis-ci.org/conda-forge/pandas-feedstock/jobs/186561921)

**Comment by wesm** [comment]
thank you!

**Comment by jorisvandenbossche** [comment]
mac osx wheel for python 3.6 is now also available (and uploaded to pypi), see https://github.com/MacPython/pandas-wheels/pull/8

**Comment by jorisvandenbossche** [comment]
And linux 3.6 wheels are now also available (https://github.com/MacPython/pandas-wheels/pull/9) and uploaded to pypi. 
Which means that now everything is uploaded
